ops/ansible/roles/bfd-server-ccs/tasks/appserver_config.yml

```diff
@@ -0,0 +1,182 @@
+---
+
+- name: Create App Server 'standalone.conf' File
+  template:
+    src: standalone.conf.j2
+    dest: "{{ data_server_dir }}/{{ data_server_container_name }}/bin/standalone.conf"
+    owner: "{{ data_server_user }}"
+    group: "{{ data_server_user }}"
+    mode: u=rwx,g=rx,o=rx
+  become: true
+  tags: 
+    - pre-ami
+
+# The AWS dev/test/prod environments aren't idempotent: the same base image 
+# will be reused and just upgraded over time. accordingly, we have to assume 
+# that the FHIR server's SSL certificate may already exist, and preserve it, if 
+# so.
+
+- name: Check for App Server SSL Keystore
+  stat:
+    path: "{{ data_server_dir }}/bluebutton-appserver-keystore.jks"
+  register: stat_appserver_keystore
+  changed_when: false
+  become: true
+  tags: 
+    - pre-ami
+
+- name: Create App Server SSL Keystore
+  command: "keytool -genkeypair -alias server {{ data_server_ssl_server_genkeypair_args }} -keystore {{ data_server_dir }}/bluebutton-appserver-keystore.jks -storepass changeit -keypass changeit"
+  when: stat_appserver_keystore.stat.isreg is not defined
+  become: true
+  become_user: "{{ data_server_user }}"
+  no_log: true  # Ensure that the key args aren't logged if this command fails.
+  tags: 
+    - pre-ami
```

Comment:
Not for this PR, but we will need to use the same process for a stable keystore as we are planning to use for the `vault.password` file.